generators/app/package.json

```diff
@@ -2,18 +2,22 @@
   "name": "polymer-starter-kit-custom-build",
   "version": "0.0.0",
   "description": "A starting point for Polymer 1.0 custom build apps",
-  "license": "http://polymer.github.io/LICENSE.txt",
+  "license": "BSD-3-Clause",
   "scripts": {
-    "build": "gulp"
+    "lint": "eslint . --ext js,html; exit 0;",
```

Comment:
I copied from PSK. But... I already create a PR about that:
https://github.com/PolymerElements/polymer-starter-kit/pull/954